charts/kafka-topic-replicator/container/pyproject.toml

```
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "kafka-topic-replicator"
+version = "0.1.0"
+description = "An application that replicates records from one Kafka topic to another with a configurable delay"
+authors = ["Your Name <you@example.com>"]
```

No author is better than non-real author or perhaps it should be `Team DT <team-dt@energinet.dk`. Thoughts?🤔 
```suggestion
authors = []
```